core/modules/installer/installer.browser.inc

```diff
@@ -413,7 +411,7 @@ function installer_browser_get_destination_after_install() {
   $theme = in_array('theme', $types);
   $module = in_array('module', $types);
 
-  $layout_message = t('Installation finished successfully. All newly-added layout templates have been enabled and are available for use in your layouts. You can manage them on the <a href="@link">Layout templates</a> page.', array('@link' => 'admin/structure/layouts/settings'));
+  $layout_message = t('Installation finished successfully. All newly-added layout templates have been enabled and are available for use in your layouts. You can manage them on the <a href="@link">Layout templates</a> page.', array('@link' => '/admin/structure/layouts/settings'));
```

Comment:
The [recommended way](https://github.com/backdrop/backdrop/blob/a07bea1362b7c9d9fb4c67535ab5508617c19696/core/includes/common.inc#L2852) is to use function `url()` to build the link (this allows hooks to run on the link). There are several examples of this usage throughout the codebase (although there are also some examples of your approach).

```suggestion
  $layout_message = t('Installation finished successfully. All newly-added layout templates have been enabled and are available for use in your layouts. You can manage them on the <a href="@link">Layout templates</a> page.', array('@link' => url('admin/structure/layouts/settings')));
```

Reply:
Of course 🤦🏼 ...done. Thanks @argiepiano 🙏🏼